settings/base.py

```diff
@@ -20,25 +25,32 @@ def path(*args):
 )
 
 MANAGERS = ADMINS
+# avoids a warning from django
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL. See travis.py in this directory
 # for an example if you'd like to run MySQL locally, and add that to your
 # local.py.
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'basket.db',
-    }
+    'default': config('DATABASE_URL',
+                      default='sqlite:///basket.db',
+                      cast=dj_database_url.parse),
 }
+if DATABASES['default']['ENGINE'] == 'django.db.backends.mysql':
+    DATABASES['default']['OPTIONS'] = {'init_command': 'SET storage_engine=InnoDB'}
 
-ALLOWED_HOSTS = [
-    '.allizom.org',
-    'basket.mozilla.com',
-    'basket.mozilla.org',
-]
-SESSION_COOKIE_SECURE = True
-CSRF_COOKIE_SECURE = True
+CACHES = {
+    'default': config('CACHE_URL',
+                      default='locmem://',
+                      cast=django_cache_url.parse)
```

Comment:
> Have you checked out django-cache-url? It seems quite good, and supports django-redis which I plan on using in the deis deploy.
> And for multiple servers you just have to comma separate them. e.g. memcached://server1:11211,server2:11211

I looked at it before and liked it, but somehow I missed the option to specify multiple servers with commas. Nice.